### DISCUSSION TITLE
2006/09/29/%E5%85%B6%E5%AE%83%E5%B8%B8%E7%94%A8%E8%B0%83%E8%AF%95%E5%B7%A5%E5%85%B7%E5%92%8C%E6%96%B9%E6%B3%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#117] [Announcements]
# 2006/09/29/%E5%85%B6%E5%AE%83%E5%B8%B8%E7%94%A8%E8%B0%83%E8%AF%95%E5%B7%A5%E5%85%B7%E5%92%8C%E6%96%B9%E6%B3%95

除了断点和探针这两种最常用的调试工具外，我们也经常要借助一些其它的工具和方法来找到程序的问题所在。

https://blog.qizhen.xyz/2006/09/29/%E5%85%B6%E5%AE%83%E5%B8%B8%E7%94%A8%E8%B0%83%E8%AF%95%E5%B7%A5%E5%85%B7%E5%92%8C%E6%96%B9%E6%B3%95

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Zhang Ping
2007年07月7日 11:31 编辑
现在只能仰视

回复
Mavis
2006年09月30日 14:09 编辑
仰视一下
